plugins/samson_ledger/test/initializers/ledger_test.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+require_relative "../test_helper"
+
+SingleCov.covered! file: 'plugins/samson_ledger/config/initializers/samson_ledger/client.rb'
+
+describe 'Ledger::Client' do
+  before do
+    stub_request(:post, "https://foo.bar/api/v1/events")
+  end
+  after do
+    ENV.delete("LEDGER_BASE_URL")
+    ENV.delete("LEDGER_TOKEN")
+  end
+
+  describe "plugin enabled" do
+    around do |test|
+      ENV.delete("LEDGER_BASE_URL")
+      ENV.delete("LEDGER_TOKEN")
+      test.call
+    end
+    it "is not enabled" do
+      Ledger::Client.plugin_enabled?
+    end
+  end
+
+  describe "cannot create client w/o a token" do
```

Comment:
I think `describes` with a single `it` add unnecessary complexity ... also start with valid and the remove single thing

```Ruby
describe ".plugin_enabled?" do
  before do
   ENV['x'] = 1
  end

  it "is enabeld" do
   assert enabled?
  end

  it "is disabled without x" do
   ENV.delete 'x'
   refute enabled?
  end
end
  
end
```